pkg/account/deployer/client.go

```diff
@@ -213,7 +227,7 @@ func (d *accountManagementClient) updatePermissions(ctx context.Context, groupId
 			return fmt.Errorf("unsupported permission %q. Must be one of: %v", p.PermissionName, d.supportedPermissions)
 		}
 	}
-	resp, err := d.client.PermissionManagementAPI.AddGroupPermissions(ctx, d.accountInfo.AccountUUID, groupId).PermissionsDto(permissions).Execute()
+	resp, err := d.client.PermissionManagementAPI.OverwriteGroupPermissions(ctx, d.accountInfo.AccountUUID, groupId).PermissionsDto(permissions).Execute()
```

Comment:
changed to `OverwriteGroupPermissions` to also cover the removal of permissions.